zebra-rpc/src/methods/get_block_template_rpcs/types/default_roots.rs

```diff
@@ -1,18 +1,30 @@
 //! The `DefaultRoots` type is part of the `getblocktemplate` RPC method output.
 
+use zebra_chain::block::{
+    merkle::{self, AuthDataRoot},
+    ChainHistoryBlockTxAuthCommitmentHash, ChainHistoryMmrRootHash,
+};
+
 /// Documentation to be added in #5452 or #5455.
 #[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
 pub struct DefaultRoots {
-    /// Add documentation.
+    /// The merkle root of the transaction IDs in the block.
     #[serde(rename = "merkleroot")]
-    pub merkle_root: String,
+    #[serde(with = "hex")]
+    pub merkle_root: merkle::Root,
+
     /// Add documentation.
     #[serde(rename = "chainhistoryroot")]
-    pub chain_history_root: String,
-    /// Add documentation.
+    #[serde(with = "hex")]
+    pub chain_history_root: ChainHistoryMmrRootHash,
+
+    /// The merkle root of the authorizine data hashes of the transactions in the block.
```

Comment:
```suggestion
    /// The merkle root of the authorizing data hashes of the transactions in the block.
```

There is a typo but also i am not sure if this description means the same as the one that can be found in https://zcash.github.io/rpc/getblocktemplate.html for the `authdataroot` field.

Reply:
Yea, it is the same.